File: XString40.cpp
#include "pt4.h"
using namespace std;

void Solve()
{
    Task("XString40");
    string s;
    pt >> s;
    int position_1 = s.find(" ");
    int position_2 = s.rfind(" ");
    s.erase(s.begin() + position_2, s.end());
    s.erase(0, position_1+1);
    pt << s;
}

File: XString40.cs
// File: "XString40"
using PT4;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace PT4Tasks
{
    public class MyTask : PT
    {
        public static void Solve()
        {
            Task("XString40");
            string s = GetString();
            int position_1 = s.IndexOf(" ");
            int position_2 = s.LastIndexOf(" ");
            if (position_1 == position_2)
            {
                s = "";
            }
            else
            {
                s = s.Remove(position_2, s.Length - position_2);
                s = s.Remove(0, position_1 + 1);
            }
            Put(s);
        }
    }
}
